packages/utils/src/http.ts

```diff
@@ -0,0 +1,26 @@
+import axios, { AxiosError } from 'axios'
+
+export class HttpError {
+  constructor(public msg: string, public httpStatus: number) {}
+}
+
+export async function post(url: string, json_data: string): Promise<string> {
+  try {
+    let response = await axios.post(url, json_data, {
+      timeout: 30_000,
+      maxRedirects: 3,
+      headers: {
+        'Content-Type': 'application/json'
```

Comment:
Does axios set other headers itself? If not we should at least also set `Accept-Encoding` and `Accept-Content`.

Reply:
Added to be explicit.